web/src/gamesShared/definitions/customization.ts

```diff
@@ -0,0 +1,17 @@
+import { GameMode } from './mode';
+
+export interface GameCustommizationState {
+  quick?: unknown;
+  full?: unknown;
+}
+
+export interface GameCustomizationProps {
+  mode: GameMode;
+  currentValue: unknown;
+  onChange: (result: unknown) => void;
+}
+
+export interface GameCustomization {
+  renderQuick?: (args: GameCustomizationProps) => JSX.Element | null;
```

Comment:
You may want to use `React.ReactNode`
![image](https://user-images.githubusercontent.com/304383/104136722-e729a300-5365-11eb-8c11-3ba17fec6aaf.png)
```ts
renderQuick: (args: GameCustomizationProps) => React.ReactNode;
```

Reply:
Good call. Done.